### PR TITLE
Update subkey infos inserting

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: internal/data/kv_pair.go [hunk 1]
```diff
@@ -1032,11 +1032,17 @@ func (rp kvPairRepo) CreateMetadataKvPairs(ctx context.Context, checkInfo biz.Ch
...
 			}
 			if len(subKeys) > 0 {
-				if err := tx.Model(SubKeyInfo{}).WithContext(ctx).Clauses(clause.OnConflict{
-					Columns:   []clause.Column{{Name: "pub_key"}, {Name: "lock_hash"}},
-					UpdateAll: true,
-				}).Create(subKeys).Error; err != nil {
-					return err
+				for _, subKey := range subKeys {
+					var oldSubkey SubKeyInfo
+					err := tx.Model(SubKeyInfo{}).WithContext(ctx).Where("lock_hash = ? and pub_key = ? and credential_id = ?", subKey.LockHash, subKey.PubKey, subKey.CredentialId).First(&oldSubkey).Error
```
Comment (shaojunda) [inline internal/data/kv_pair.go:1037]: maybe we can creat an index on the `lock_hash`

Comment (duanyytop) [reply internal/data/kv_pair.go:1037]: The `lock_hash` currently has duplicate data

Comment (duanyytop) [reply internal/data/kv_pair.go:1037]: Updated https://github.com/nervina-labs/cota-syncer/pull/35/commits/a59e763b5273d2ed869d42964b216c2d147ac1e2

File: internal/db/migrations/000020_add_subkey_info_index.up.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+ALTER TABLE sub_key_infos ADD INDEX uc_sub_key_infos_on_lock_hash(lock_hash);
```
Comment (shaojunda) [inline internal/db/migrations/000020_add_subkey_info_index.up.sql:1]: `uc_sub_key_infos_on_lock_hash(lock_hash)` -> `idx_sub_key_infos_on_lock_hash(lock_hash);`

Comment (duanyytop) [reply internal/db/migrations/000020_add_subkey_info_index.up.sql]: done

**Review by shaojunda — APPROVED** [review]

**Pull request merged**